packages/viewer/src/components/viewer-pin-tool/viewer-pin-tool.css

```diff
@@ -0,0 +1,22 @@
+:host {
+  /**
+   * @prop  --viewer-annotations-pin-label-line-color: A CSS color that
+   * specifies the color of the label line
+   */
+   --viewer-annotations-pin-dot-color: var(--blue-600);
+
+     /**
+   * @prop  --viewer-annotations-pin-label-line-color: A CSS color that
+   * specifies the color of the label line
+   */
+   --viewer-annotations-pin-dot-border-color: var(--white);
+
+  font-size: 0.75rem;
```

Comment:
Are these values needed for the tool here?

Reply:
Will remove